scss/field_widget.scss

```diff
@@ -0,0 +1,29 @@
+fieldset:not(.fieldgroup)[class*="MuiOutlinedInput"] {
+  position: absolute;
+  margin: 0;
+  padding: 0;
+  background-color: transparent;
+}
+
+.MuiInputLabel-outlined {
+  &.MuiInputLabel-shrink {
+    background: #fff;
+  }
+}
+
+legend {
+  &.MuiFormLabel-root {
+    margin-bottom: 10px;
+  }
+}
+
+.react-paragraphs-fieldset {
+  legend {
+    text-transform: none;
+    letter-spacing: 0;
+  }
+}
+
+.form-modal-root {
+  z-index: 1100 !important;
```

Comment:
does it have to be important?

Reply:
Yeah unfortunately.  It gets set via javascript when drupal opens the modal

Reply:
i correct myself. i was able to move it to the react component